doc/releases/changelog-0.12.0.md

```diff
@@ -2,49 +2,86 @@
 
 <h3>New features since last release</h3>
 
-* Added integration with PennyLane's experimental python compiler based on xDSL.
-  This allows developers and users to write xDSL transformations that can be used with Catalyst.
-  [(#1715)](https://github.com/PennyLaneAI/catalyst/pull/1715)
-
-* A new compilation pass called :func:`~.passes.ppr_to_ppm` has been added to Catalyst
-  to decompose Pauli product rotations (PPRs), :math:`\exp(-iP_{\{x, y, z\}} \theta)`, into
-  Pauli product measurements (PPMs). Non-Clifford PPR (:math:`\theta = \tfrac{\pi}{8}`) requires
-  the consumption of a magic state, while Clifford PPR (:math:`\theta = \tfrac{\pi}{4}`) will not.
-  The methods are implemented as described in [arXiv:1808.02892](https://arxiv.org/abs/1808.02892v3).
+* A new compilation pass called :func:`~.passes.ppm_compilation` has been added to Catalyst to
+  transform Clifford+T gates into Pauli Product Measurements (PPMs) using just one transform.
```

Comment:
```suggestion
  transform Clifford+T gates into Pauli Product Measurements (PPMs) using just one transform, allowing for 
  exploring representations of programs in a new paradigm in logical quantum compilation.
```

@dime10 how's that?

Reply:
That's good! Should we also mention the program compiled this way are non-executable?